frontend/src/shared/components/left-sidebar/left-sidebar.component.html

```diff
@@ -10,39 +10,39 @@
     <ul>
       <li>
         <div class="menu-item">
-          <a href="/board">
+          <a href="project/board">
```

Comment:
could use `routerLink` instead of `href` for consistency

Reply:
Valuable  comment
Href is the basic attribute provided by Html to navigate through pages which reloads the page on click.
routerLink is the attribute provided by angular to navigate to different components without reloading the page.
https://stackoverflow.com/questions/58081360/anchor-href-vs-angular-routerlink#:~:text=Href%20is%20the%20basic%20attribute,components%20without%20reloading%20the%20page.

Reply:
Changed href to router link